docs/specification.md

```diff
@@ -1,12 +1,43 @@
-# Homebase Lite Spec Draft
+<!--
+   - SPDX-FileCopyrightText: 2022 Tezos Commons
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TC
+   -->
+
+# Homebase Lite Spec
+
+This spec is for the proposed Tezos smart contract to be used as the on-chain component of a new lightweight Homebase tool.
+
+The contract is under active development, hence changes to the spec are possible.
+
+<!-- TOC START min:1 max:3 link:true asterisk:false update:true -->
```

Comment:
:thinking: I haven't seen this before, is it generated or used by some automated tool?

Reply:
https://github.com/nok/markdown-toc

Reply:
(it's an editor plugin)

Reply:
Well, this looks nice.

Do you mind adding a comment (in a separate PR, that's fine) to mention its name here, for future reference?